README.rst

```diff
@@ -50,31 +48,29 @@ Also included are SQL drivers:
 |Hibernate5        |Java 8 or 11         |
 +------------------+---------------------+
 
-Of course, these clients require a NuoDB database to be running.  If you don't
-have NuoDB_ available yet, you can use the `NuoDB Community Edition`_.
+The use of the client drivers require a running NuoDB database.  If you don't
```

Comment:
Again, there's more here than just drivers.  We shouldn't unnecessarily restrict these statements to the drivers else users will wonder about the non-driver contents.

Reply:
removed "drivers" in this location.